hedera-mirror-rest/server.js

```diff
@@ -118,6 +118,9 @@ app.set('port', port);
 app.set('query parser', requestQueryParser);
 
 serveSwaggerDocs(app);
+if (isTestEnv()) {
+  validateSwaggerDocs(app);
```

Comment:
It's not really related to Swagger or docs:

```suggestion
  openApiValidator(app);
```

Reply:
Yeah, I like that better, fixed.